docker/README.md

```diff
@@ -0,0 +1,60 @@
+Docker Support
```

Comment:
This is great, thanks for adding docker support! We aim to have tests with every PR that adds functionality. Do you think the 'docker build' command is something that should/could be added as a devops tests? How long does it take to run?

Reply:
It takes 16min to finish the build of a Docker image. We can probably have a specific build pipeline that runs testing on the Docker build. It may not be triggered by PR as I don't expect the Dockerfile will be updated with high frequency. What do you think?

Reply:
@jiata Adding JS to this thread. 16min isn't too bad, and acceptable for our over-night devops pipeline. See the tests under tests\smoke\test_azureml_notebooks.py, they also take up to 30min each. 
But agree, if the test only verifies that the docker build commend worked, then this test might not be needed. Ideally we'd also want to test that CVBP runs successfully inside the docker container, but not sure that's doable?

Reply:
@yueguoguo are the other repos doing or planning on doing any sort of testing around the docker containers? Also, unrelated, are the othee repos hosting build docker containers for download?

Reply:
@PatrickBue reco does not have testing pipeline for the Docker file, and nlp repo does not have Docker support at the moment. It makes sense for having testing for the Docker things, but not sure if it is sensible to run all the testing. 

@jiata to run test within a Docker container, see [here](https://github.com/microsoft/recommenders/tree/master/docker#running-tests-with-docker).